PocketKit/Sources/PocketKit/Item/ItemListView.swift

```diff
@@ -42,6 +42,11 @@ struct ItemListView: View {
                     .swipeActions {
                         favoriteButton(for: item)
                             .tint(Color(.branding.amber3))
+                        Button {
+                            source.delete(item: item)
+                        } label: {
+                            Label("Delete", systemImage: "trash")
+                        }.tint(.red)
```

Comment:
I kind of like how the favorite button is extracted into its own convenience function, and feel like maybe it would "clean things up" a little, here, even though it's quite obvious what's being done 🙃